proxy/envoy/config.go

```diff
@@ -267,6 +268,21 @@ func buildOutboundHTTPRoutes(instances []*model.ServiceInstance, services []*mod
 					routes = append(routes, buildDefaultRoute(cluster))
 				}
 
+				if service.External {
+					for _ , route := range routes {
+						for _ , cluster := range route.clusters {
+							cluster.ServiceName = ""
```

Comment:
Yes I was debating about whether to use a default URL from MeshConfig or pointing it to SDS.  I chose the meshconfig option because I figured otherwise I would have to either store the egress model.Service or cache it somehow so I could reliably generate a `ServiceName` for the cluster